lib/puppet/provider/registry_value/registry.rb

```diff
@@ -36,11 +40,11 @@ def exists?
       hive.open(subkey, Win32::Registry::KEY_READ | access) do |reg|
         FFI::Pointer.from_string_to_wide_string(valuename) do |valuename_ptr|
           status = Puppet::Util::Windows::Registry.RegQueryValueExW(reg.hkey, valuename_ptr,
-            FFI::MemoryPointer::NULL, FFI::MemoryPointer::NULL,
-            FFI::MemoryPointer::NULL, FFI::MemoryPointer::NULL)
+                                                                    FFI::MemoryPointer::NULL, FFI::MemoryPointer::NULL,
+                                                                    FFI::MemoryPointer::NULL, FFI::MemoryPointer::NULL)
 
-          found = status == 0
-          raise Win32::Registry::Error.new(status) if !found
+          found = status.zero?
+          raise Win32::Registry::Error, status unless found
```

Comment:
Is this actually going to work? The command after `Error` but before the parameter makes me worry that `status` might not be interpreted as the parameter to the `new()` method on `Error`.

Reply:
I don't really understand yet how this syntax works, but it's what Rubocop wants and it does seem to work, so it looks good to me I guess.